---
title: "Research"
date: 2019-10-13
showDate: false
---

<!-- ## Accuracy and reproducibility of microbiome measurements -->
## Experimental bias in metagenomics measurements

Microbiome measurements made via community sequencing (shotgun metagenomics or
marker-gene sequencing) are currently inaccurate because existing experimental
protocols are biased to measure some species 10-1000X more efficiently than
others. As the different protocols used in different experiments enrich for
different species, the measurements from different experiments are
quantitatively incomparable. My current research aims to 

- develop new methods to characterize bias from control experiments
- evaluate the effects of bias on microbiome analyses and diagnostics
- develop calibration methods to correct bias, making microbiome measurements
  accurate or at least quantitatively comparable across labs

This work is being carried out under the supervision of [Ben
Callahan](https://callahanlab.cvm.ncsu.edu/team/) at NC State and in
collaboration with [Amy Willis and David
Clausen](http://statisticaldiversitylab.com/team) at the University of
Washington. Our first effort was recently [published in
eLife](https://elifesciences.org/articles/46923); you can read a short summary
[here](https://twitter.com/mikemc423/status/1177287267657768960).

## _C. difficile_ ecology and pathogenesis

I provide bioinformatic support to efforts in the lab of [Casey
Theriot](https://theriotlab.org/) to study the ecology and pathogenesis of the
gut pathogen _C. difficile_. Projects I am involved in include characterizing
the prevalence of _C. difficile_ in domestic animals and understanding the role
of the gut microbiota in preventing _C. difficile_ colonization and disease in
domestic animals and in experimental mouse models.

## Evolutionary dynamics

My PhD research used probabilistic mathematical modeling to study how spatial
(geographic) structure affects the time it takes for a population to acquire a
complex adaptation.
